PUSHDOWN.md

```diff
@@ -0,0 +1,287 @@
+| Filters                                                                    | [WHERE clause](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-where.html)                                                                                                |
+| In/InSet                                                                   | [IN clause](https://spark.apache.org/docs/3.3.0/sql-ref-null-semantics.html#innot-in-subquery-)                                                                                         |
+| Having                                                                     | [HAVING clause](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-having.html)                                                                                              |
+| Joins (Inner, LeftOuter, RightOuter, FullOuter, Cross, LeftSemi, LeftAnti) | [JOIN clause](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-join.html#content)                                                                                          |
+| Limits                                                                     | [LIMIT clause](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-limit.html)                                                                                                |
+| Projections                                                                | [SELECT clause](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select.html)                                                                                                     |
+| Sorts (ORDER BY)                                                           | [ORDER BY](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-orderby.html) and [SORT BY](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-sortby.html) clauses |
+| Union and Union All                                                        | [UNION and UNION ALL clauses](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-setops.html#union)                                                                          |
+| Window functions and Windowing clauses                                     | [WINDOW clause](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-qry-select-window.html#content)                                                                                      |
+
+
+## Caveats/Limitations
+
+* Because the translation in Query Pushdown from the Spark LogicalPlan to SQL requires almost a one-to-one translation of Spark SQL operators to BigQuery expressions, not all of Spark SQL operators can be pushed down. When pushdown fails, the connector falls back to a less-optimized execution plan. The unsupported operations are instead performed in Spark.
```

Comment:
s/falls back to a less-optimized execution plan/falls back to the original execution plan/

Reply:
Ack